go-controller/pkg/ovn/gateway_init.go

```diff
@@ -589,8 +589,12 @@ func (oc *Controller) addNodeLocalNatEntries(node *kapi.Node, mgmtPortMAC string
 
 	mgmtPortName := types.K8sPrefix + node.Name
 	stdout, stderr, err := util.RunOVNNbctl("--if-exists", "lr-nat-del", types.OVNClusterRouter,
-		"dnat_and_snat", externalIP.String(), "--",
-		"lr-nat-add", types.OVNClusterRouter, "dnat_and_snat",
+		"dnat_and_snat", externalIP.String())
```

Comment:
really we should check if the lr-nat is wrong before we remove it and add it, but this code is going away anyway soon, and this was the previous behavior anyway, so not worried about it